hardware/hardware.mk

```diff
@@ -145,25 +151,31 @@ $(HARDWARE_PATH)/%.v: $(HARDWARE_PATH)/%.rom
                bsg_bladerunner_configuration > $@
 
 
-# the default case for memory config features
-CL_MANYCORE_MEMSYS_FEATURE_DMA   := 0
-CL_MANYCORE_MEMSYS_FEATURE_CACHE := 1
-
-# do we use dramsim3?
-ifneq ($(filter dramsim3, $(subst _, ,$(CL_MANYCORE_MEM_CFG))),)
-CL_MANYCORE_MEMSYS_FEATURE_DMA   := 1
-CL_MANYCORE_MEMSYS_FEATURE_CACHE := 1
-endif
-
-# do we use infinite memory?
-ifneq ($(filter infinite, $(subst _, ,$(CL_MANYCORE_MEM_CFG))),)
-CL_MANYCORE_MEMSYS_FEATURE_DMA   := 1
-CL_MANYCORE_MEMSYS_FEATURE_CACHE := 0
-endif
+include $(HARDWARE_PATH)/memsys.mk
+
+.PHONY: charvtobin-debug
+charvtobin-debug:
+	@echo $(CL_MANYCORE_MEMSYS_ID) = $(call charv2bin, $(CL_MANYCORE_MEMSYS_ID))
```

Comment:
This is probably useful to leave in, but just to make sure, this was your intent?

Reply:
It was not - I'm going to remove it.